tests/robot-tests/run_tests.py

```diff
@@ -186,9 +185,6 @@
 for env_var in required_env_vars:
     assert os.getenv(env_var) is not None, f"Environment variable {env_var} is not set"
 
-if args.slack_webhook_url:
```

Comment:
This code was setting

```
if args.slack_webhook_url:
    os.environ["SLACK_WEBHOOK_URL"] = args.slack_webhook_url
```

and the only place I could see `SLACK_WEBHOOK_URL` being used was in `slack.py` :

```
self.alert_webhook_url = os.getenv("SLACK_WEBHOOK_URL")
```

which has been removed, and the only place I could see `self.alert_webhook_url` being used was in

```
def send_snapshot_alert(self, err_msg: str) -> None:
        slack_webhook = WebhookClient(self.alert_webhook_url)
        response = slack_webhook.send(text=err_msg)
        assert response.status_code == 200 and response.body == "ok"
        logger.info("Sent snapshot alert to #build")
```

which has also been removed.

The slack notifications for reports sent in `slack.py` are using different variables i.e.
`os.getenv("SLACK_BOT_TOKEN")` and
`os.getenv("SLACK_TEST_REPORT_WEBHOOK_URL")`.

We might want to think about adding code in so that these variables for the report notifications can be set securely as I believe they are currently only working through environment variables set in the pipeline rather than through Key Vault.

Reply:
Ah yeah, that's right. Yep, think they should be moved to key vault at some point